tests/indices/keyword_table/test_base.py

```diff
@@ -65,8 +69,14 @@ def test_build_table(
     "gpt_index.indices.keyword_table.simple_base.simple_extract_keywords",
     mock_extract_keywords,
 )
+@patch.object(LLMPredictor, "total_tokens_used", return_value=0)
 @patch.object(LLMPredictor, "__init__", return_value=None)
-def test_insert(_mock_init: Any, _mock_predict: Any, documents: List[Document]) -> None:
+def test_insert(
+    _mock_init: Any,
+    _mock_total_tokens_used: Any,
```

Comment:
I have to add this to every test that does `@patch.object(LLMPredictor, "__init__",...` otherwise pytest complains that `_total_tokens_used` doesn't exist as a class attribute

Reply:
ah yeah - i might refactor the mocks in a later PR to better reuse patch decorators but not for this pr!

Reply:
sounds good! created an issue for this: https://github.com/jerryjliu/gpt_index/issues/100